README.sle11-branch

```diff
@@ -0,0 +1,15 @@
+the test cases and needles need to be up to dayte anyhow and change at
+a very high pace.  So we pull in upstream via a separately cloned repo
+(on the master branch) and just symlink to that, like so:
+
+git clone https://github.com/os-autoinst/os-autoinst-distri-opensuse opensuse-testcases
+
```

Comment:
no, we don't. We just symlink to /opensuse 

I'm fine with having documentation in here, but no lies please.